packages/cli/generation/ir-generator/src/converters/services/convertHttpService.ts

```diff
@@ -371,19 +377,24 @@ export function getHeaderName({ headerKey, header }: { headerKey: string; header
     };
 }
 
-function convertTransportToEncoding(transport: Transport): Encoding {
-    switch (transport.type) {
-        case "http":
-            return {
-                json: {},
-                proto: undefined
-            };
-        case "grpc":
-            return {
-                json: undefined,
-                proto: {}
-            };
-        default:
-            assertNever(transport);
+function convertEndpointSourceToEncoding(source: RawSchemas.SourceSchema | undefined): Encoding {
```

Comment:
Can we rename this `convertSourceToEncoding` and use it for both the service and endpoint? It looks like we have access to the `SourceSchema` in both instances, so we may as well keep it consistent.

Reply:
Encoding is only in service at the moment. I can add it to the endpoint, too, but I don't know if we even need this property. It's not used anywhere AFAIK.